tests/basic/tests/http.py

```diff
@@ -149,3 +149,10 @@ def test_private_cache_control(self):
 
         self.assertEqual(cache_control, set(["s-maxage=3600", "max-age=3600", "private"]))
         self.assertTrue('"johndoe"' in response.read().decode('utf-8'))
+
+    def test_filter_in(self):
+        connection = self.get_connection()
+        connection.request('GET', '/api/v1/notes/1/filterin/', headers={'Accept': 'application/json'})
```

Comment:
I'd like to see this actually pass the filter in as part of the query, to show it splitting the csv list of IDs as well.